Source/Core/DolphinQt/Config/Mapping/WiimoteEmuGeneral.cpp

```diff
@@ -89,6 +90,9 @@ void WiimoteEmuGeneral::OnAttachmentChanged(int extension)
   GetParent()->ShowExtensionMotionTabs(extension == WiimoteEmu::ExtensionNumber::NUNCHUK);
 
   m_extension_widget->ChangeExtensionType(extension);
+
+  // Update TAS Input window
+  Host_UpdateWiiTASInputExt(GetPort(), extension);
```

Comment:
This is a bit weird. Why do we have a host function that's only called from DolphinQt and only implemented in DolphinQt? You could probably just use a Qt signal directly instead of going through Host.h.

Reply:
When you put it like that I realize this was a silly approach. Will fix later today